Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -640,35 +641,60 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
         }
     }
 
-    private func addDependency(to dependency: ResolvedTarget.Dependency, in pifTarget: PIFTargetBuilder, linkProduct: Bool) {
+    private func addDependency(
+        to dependency: ResolvedTarget.Dependency,
+        in pifTarget: PIFTargetBuilder,
+        linkProduct: Bool
+    ) {
         switch dependency {
-        case .target(let target, _):
-            addDependency(to: target, in: pifTarget, linkProduct: linkProduct)
-        case .product(let product, _):
-            addDependency(to: product, in: pifTarget, linkProduct: linkProduct)
+        case .target(let target, let conditions):
+            addDependency(
+                to: target,
+                conditions: conditions,
```

Comment:
Purely nit-pick, but I think it would read better with conditions as third argument.

Reply:
👍